src/background.js

```diff
@@ -116,9 +121,22 @@ const rollout = {
     await browser.study.setup(baseStudySetup);
     browser.runtime.onMessage.addListener((...args) => this.handleMessage(...args));
   },
+  async showTab() {
+    const tabs = await this.findStudyTabs();
+    if (tabs.length) {
+      browser.tabs.update(tabs[0].id, {
+        active: true
+      });
+    } else {
+      browser.tabs.create({
+        url: STUDY_URL
+      });
+    }
+  },
   async onReady() {
     const studyInfo = await browser.study.getStudyInfo();
     if (!studyInfo.isFirstRun) {
+      this.setupAlarm();
```

Comment:
I'm not following this, why is the alarm only setup if it's not the first run?

Reply:
As later code calls it on showing of the banner, however this is just to cater for sending the pings when the user has restarted.